### ISSUE TITLE
Placeholder API Integration

### ISSUE THREAD

**Issue by philip-ulrich** [yariplus/nodebb-plugin-minecraft-integration#83]
Hey man,

I have a feature request I've been mulling over. I think PlaceholderAPI would be an awesome thing to integrate into the plugin. This would allow people to use thousands of variables to send to nodebb. On the web side of the house, you could make these nodebb Placeholders (those are a thing, right? And then either offer a way to put them in posts/on the site or let us know how we can use them on the site. This would allow for lots of cool things to be shown on the forums.

**Comment by yariplus** [comment]
It looks like a good addition, but I am turned off by it's non-permissive license.

Let me mull over it for a while.

**Comment by philip-ulrich** [comment]
Yeah. The dev is not necessarily into the whole "open source" thing unfortunately. I think it'd be beneficial though.

**Comment by yariplus** [comment]
I'm going to do this, but it won't be a priority, unless I got more requests. This is mainly because I want to keep most features universal, and no sponge equivalent exists, or is ever likely to exist, due to the plugin's closed-source.

**Comment by philip-ulrich** [comment]
Sure thing. I completely understand.

If this guy keeps the efforts up, you might be able to use this for sponge: https://forums.spongepowered.org/t/api-placeholderapi-v1-0-snapshot/11729

I'm not sure if this is going to happen though. The last commits were a couple months ago.

**Comment by yariplus** [comment]
Just reaffirming this definitely going to happen. Just have not had time lately with other projects.
